tools/test/examples/examples_cmake.json

```diff
@@ -0,0 +1,133 @@
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : ["WIO_3G"],
+        "toolchains" : [],
+        "exporters": [],
+        "compile" : true,
+        "export": true,
+        "test" : false,
+        "auto-update" : true
+    },
+    {
+        "name": "mbed-os-example-lorawan",
+        "github":"https://github.com/ARMmbed/mbed-os-example-lorawan",
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : ["K64F"],
+        "toolchains" : [],
+        "exporters": [],
+        "compile" : true,
+        "export": false,
+        "test" : false,
+        "auto-update" : true
+    },
+    {
+        "name": "mbed-os-example-devicekey",
+        "github":"https://github.com/ARMmbed/mbed-os-example-devicekey",
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : ["K66F"],
+        "toolchains" : [],
+        "exporters": [],
+        "compile" : true,
+        "export": true,
+        "test" : true,
+        "baud_rate": 9600,
+        "compare_log": ["mbed-os-example-devicekey/tests/devicekey.log"],
+        "auto-update" : true
+    },
+    {
+        "name": "mbed-os-example-mbed-crypto",
+        "github":"https://github.com/ARMmbed/mbed-os-example-mbed-crypto",
+        "sub-repo-example": true,
+        "subs": ["getting-started"],
+        "features" : [],
+        "targets" : ["K64F"],
+        "toolchains" : [],
+        "exporters": [],
+        "compile" : true,
+        "export": true,
+        "test" : true,
+        "baud_rate": 9600,
+        "compare_log": ["mbed-os-example-mbed-crypto/tests/getting-started.log"],
+        "auto-update" : true
+    },
+    {
+        "name": "mbed-os-example-nfc",
+        "github": "https://github.com/ARMmbed/mbed-os-example-nfc",
+        "sub-repo-example": true,
+        "subs": [
+            "NFC_EEPROM",
+            "NFC_SmartPoster"
+        ],
+        "features" : [],
+        "targets" : ["NUCLEO_F401RE"],
+        "toolchains" : [],
+        "exporters": [],
+        "compile" : true,
+        "export": true,
+        "test" : true,
+        "baud_rate": 9600,
+        "compare_log": [
+            "mbed-os-example-nfc/tests/EEPROM.log",
+            "mbed-os-example-nfc/tests/SmartPoster_noShield.log"
+        ],
+        "auto-update" : true
+    },
+    {
+        "name": "mbed-os-example-kvstore",
+        "github":"https://github.com/ARMmbed/mbed-os-example-kvstore",
+        "sub-repo-example": false,
+        "subs": [],
+        "features" : [],
+        "targets" : ["K64F", "DISCO_L475VG_IOT01A"],
+        "toolchains" : [],
+        "exporters": [],
+        "compile" : true,
+        "export": true,
+        "test" : true,
+        "baud_rate": 9600,
+        "compare_log": ["mbed-os-example-kvstore/tests/kvstore.log"],
+        "auto-update" : true
+    }
```

Comment:
Should we have the BLE example in this list, too?

Reply:
BLE example added